bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -304,9 +428,28 @@ Notes:
 - Firmware updates are currently provided at [Velux download area](https://updates2.velux.com/).
 
 
+## How to run it under openHAB2?
```

Comment:
This section should be removed as it is not relevant for 1.x bindings that are made available in the official distro, see  https://www.openhab.org/docs/developer/legacy/compatibilitylayer.html#how-to-add-a-successfully-tested-1-x-add-on-to-the-distribution. If no feature is defined for this binding so far, please add it in this PR and installation will be straight forward, just like any other binding.

Reply:
Thanks for those pointers.

Fixed as proposed.